service/src/test/java/uk/gov/hmcts/reform/fpl/model/AddressTest.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class AddressTest {
+
+    private Address.AddressBuilder builder;
+
+    @BeforeEach
+    void setup() {
```

Comment:
Don't think this is worth having just for 2 tests, can just make the address builder local to the methods that is using it. We are not doing anything complex really so could be inlined